README.md

```diff
@@ -117,7 +117,8 @@ You can find a (way cooler) web version of this list [here](https://gbdev.github
 - [ESP8266 GB Printer](https://github.com/applefreak/esp8266-gameboy-printer) - A device that emulates the GB Printer and lets you retrieve images using WiFi.
 - [fruttenboel](http://verhoeven272.nl/fruttenboel/Gameboy/index.html) - Page with loads of information on the hardware, custom boards to interface with the console and other related projects.
 - [Game Boy hardware database](https://gbhwdb.gekkio.fi/) - Data and photos of various types of Game Boy consoles.
-- [DMG-CPU-Inside](https://github.com/furrtek/DMG-CPU-Inside) - Schematics and annotated overlay for the DMG-CPU-B chip, extracted from die photos.
+- [DMG-CPU-Inside](https://github.com/furrtek/DMG-CPU-Inside) - Schematics and annotated overlay for the DMG-CPU B chip, extracted from die photos.
+- [dmg-schematics](https://github.com/msinger/dmg-schematics) - Fixed and cleaned up schematics and overlay for the DMG-CPU B chip, remade with [KiCad](https://www.kicad.org/). Also contains [Electric VLSI](https://www.staticfreesoft.com/) library with layouts for some of the cells and memories.

```

Comment:
```suggestion
- [dmg-schematics](https://github.com/msinger/dmg-schematics) - Fixed and cleaned up schematics and overlay for the DMG-CPU B chip, remade with KiCad. Also contains an Electric VLSI library with layouts for some of the cells and memories.
```
I'd rather not have the links, since they're not directly related to the entry.

Reply:
I replaced the original link and appended to the description. Like this?